### DISCUSSION TITLE
Unable to correctly display a PDF file

### DISCUSSION THREAD

**Discussion by azt3c123** [sumatrapdfreader/sumatrapdf#4471] [General]
Hi, i've been using and installing sumatra everywhere for years, no errors, working perfectly every time, fast and reliable. 
I work with a lot of PDF files, and this ones started to arrive this week, and i don't know why, i can't see them using sumatra, but with acrobat there is no issue...

font is times new roman, so is not that....

what can i do to be able to watch them using sumatra ?

[1.pdf](https://github.com/user-attachments/files/16606111/1.pdf)

**Comment by GitHubRulesOK** [comment]
The file has been severely edited and thus fonts are generally faulty. there are lots of font related data but poorly defined and much text simply named without correct setting.

It claims it is an "editedScannedDoc" so I would suspect poor OCR ? however it looks like human intervention `/TouchUp_TextEdit` since much of the contents are missing and only 33 items left.

Whilst it works very well in Acrobat it is troublesome to edit so the blue never shows and the date is hard to refine so stripping out the bad fonts see the working file below which is now much leaner as 26 working objects.
![image](https://github.com/user-attachments/assets/66388606-1888-49bb-9a42-f27fd5dae9fb)


[4471-1-fixed.pdf](https://github.com/user-attachments/files/16606445/4471-1-fixed.pdf)


![image](https://github.com/user-attachments/assets/fae08b8a-60a6-4b68-9a60-cf796e083bf9)

**Comment by azt3c123** [comment]
thank you for the explanation, so.... there is no way sumatra can display it correctly ? it would be a shame, i would have to find another pdf viewer :(

**Reply by GitHubRulesOK** [reply]
Acrobat as an editing "Reader" simply ignores faults and replaces text as required so you can extract the replaced text for reading or re-writing but as a trimmed down variant of Acrobat PRO it will sometimes simply copy the flaws when you use "save as" (tried that with this file and it saves the flaws) or in some other cases it will totally rewrite a File on save as.

Simply run the file through a font correction here Is the simplest command  to fix fonts in bulk in a file the result is a working file for all viewers.
`gs.lnk -sDEVICE=pdfwrite -o "%cd%\2.pdf" -f "%cd%\1.pdf"`

```
Processing pages 1 through 1.
Page 1
Loading font Times New Roman,Bold (or substitute) from %rom%Resource/Font/NimbusRoman-Bold
Loading font Times New Roman (or substitute) from %rom%Resource/Font/NimbusRoman-Regular
Loading font Courier New (or substitute) from %rom%Resource/Font/NimbusMonoPS-Regular
```
![image](https://github.com/user-attachments/assets/0e1addb0-b48d-497f-8c92-70ad0527386e)

[2.pdf](https://github.com/user-attachments/files/16613210/2.pdf)

![image](https://github.com/user-attachments/assets/7c7bf221-8cd0-40a3-9591-7d6d012ab3d7)
